multi-casskop/samples/gke/terraform/cloud-dns.tf

```diff
@@ -0,0 +1,134 @@
+// Documentations support : https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/gke.md
```

Comment:
Why not just use the Kubernetes deployment files we have to create those accounts ?
We could keep Terraform for the infrastructure but not for Kubernetes content ?

Reply:
The aim of this tutorial is to focus on the Multi-CassKop deployment and usage, I don't want to add complexity and risks of mistakes. With terraform provisioning also the casskop and external dns, I ensure that the requirements to instantiate Multi-casskop will be fullfilled

Reply:
Yep except not everybody uses terraform .. up to you

Reply:
In my humble opinion, I think Alexandre adds value in the form of a terraform deployment of casskop. Also he can focus on MCK testing. I would vote for keeping it as it is

Reply:
I’m not proposing to drop it but to keep kubernetes changes at the object level
In yaml files. In short if people don’t use terraform they can’t use this code 🤷‍♂️

Reply:
Yep I know I was just thinking a complete terraform example might be interesting too. terraform is widely spread I like to believe, at least it is here. Anyway, I let you decide but thanks for your detailed reveiw on this one again!